specification/inference/unified_inference/UnifiedRequest.ts

```diff
@@ -0,0 +1,214 @@
+/*
```

Comment:
we usually try to have only the request class in the corresponding Request file, and all other types we put in the `types` folder above (this is just to make it easier to maintain)